homeassistant/components/prometheus/__init__.py

```diff
@@ -161,6 +162,15 @@ def handle_event(self, event):
         )
         metric.labels(**self._labels(state)).inc()
 
+        available_metric = self._metric(
+            "entity_available", self.prometheus_cli.Gauge, "Indicates if the entity is available (1=available,0=unavailable)"
```

Comment:
```suggestion
            "entity_available", self.prometheus_cli.Gauge, "The availability of an entity"
```

To keep in style with other comments. Also the boolean mapping should be fairly self-explanatory.